x/programs/runtime/runtime.go

```diff
@@ -45,7 +45,7 @@ type WasmRuntime struct {
 	log logging.Logger
 }
 
-func (r *WasmRuntime) Initialize(ctx context.Context, programBytes []byte, maxUnits uint64) (err error) {
+func (r *WasmRuntime) Initialize(ctx context.Context, callContext program.Context, programBytes []byte, maxUnits uint64) (err error) {
```

Comment:
nit: this is my (subjective) preference for the argument order, (passing any ctx like object either first or right after golang's `context.Context`), but overall it seems we can be more consistent